src/main/java/az/edu/turing/Entities/Human.java

```diff
@@ -29,12 +30,23 @@ public Human(String name, String surname, int year, int iq, String[][] schedule,
     }
 
     public String greetPet() {
-        return "Hello, " + family.getPet().getNickname();
+        Set<Pet> pets = family.getPets();
+        StringBuilder result = new StringBuilder();
+        for (Pet pet : pets) {
+            result.append(pet.getNickname()).append(", ");
+            result.delete(result.length() - 2, result.length());
```

Comment:
why? are you sure this line should be inside loop? Also you can handle this case in another way with if condition

Reply:
We want to greet all pets.